### PR TITLE
Do some renaming for newly-added xfb metadata

### PR DESCRIPTION
**Pull request by amdrexu**
1. Use InvalidValue to mark this value is not specified
2. Use xfbBuffer to make it consist with GLSL naming
3. Rewrite some comments to explain the map info

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 6eafdd352bf169268f29940853b68d11ad23d898
<details><summary>CTS tests (Failed: 0/207274)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi2x</summary><ul>

<li>Passed: 39104/70473 (55.5%)</li>
<li>Failed: 0/70473 (0.0%)</li>
<li>Not Supported: 31369/70473 (44.5%)</li><li>Warnings: 0/70473 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 35949/70481 (51.0%)</li>
<li>Failed: 0/70481 (0.0%)</li>
<li>Not Supported: 34532/70481 (49.0%)</li><li>Warnings: 0/70481 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx9</summary><ul>

<li>Passed: 33668/66320 (50.8%)</li>
<li>Failed: 0/66320 (0.0%)</li>
<li>Not Supported: 32652/66320 (49.2%)</li><li>Warnings: 0/66320 (0.0%)</li>
</ul></details>


</ul></details>

File: llpc/translator/lib/SPIRV/SPIRVReader.cpp [hunk 1]
```diff
@@ -7166,8 +7166,11 @@ bool SPIRVToLLVM::translate(ExecutionModel entryExecModel, const char *entryName
   // ; buffer${i}Stream is the vertex StreamId that writes to buffer i, or -1 if that particular buffer is unused (e.g.
   // StreamId in EmitVertex)
   if (enableXfb) {
-    const unsigned bufferCount = 8;
-    std::array<int, 8> xfbState{-1, 0, -1, 0, -1, 0, -1, 0};
+    static const unsigned MaxXfbBuffers = 4;
+    std::array<unsigned, 2 * MaxXfbBuffers> xfbState{SPIRVID_INVALID, 0,  // xfbBuffer[0] -> <streamId, xfbStride>
+                                                     SPIRVID_INVALID, 0,  // xfbBuffer[1] -> <streamId, xfbStride>
+                                                     SPIRVID_INVALID, 0,  // xfbBuffer[2] -> <streamId, xfbStride>
+                                                     SPIRVID_INVALID, 0}; // xfbBuffer[3] -> <streamId, xfbStride>
```
Comment (nhaehnle) [inline llpc/translator/lib/SPIRV/SPIRVReader.cpp:7173]: Should this be InvalidValue for consistency? It's not a SPIRV-ID...

Comment (amdrexu) [reply llpc/translator/lib/SPIRV/SPIRVReader.cpp:7173]: Yes, you are right. In SPIR-V translator, we have legacy problems in using SPIRVID_INVALID. It is better to always use InvalidValue to make it clear.

**Comment by amdvlk-admin** [comment]
### Test summary for commit c72f061cc4482728dfac49016afb4cc207372cc1
<details><summary>CTS tests (Failed: 0/207274)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi2x</summary><ul>

<li>Passed: 39104/70473 (55.5%)</li>
<li>Failed: 0/70473 (0.0%)</li>
<li>Not Supported: 31369/70473 (44.5%)</li><li>Warnings: 0/70473 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 35949/70481 (51.0%)</li>
<li>Failed: 0/70481 (0.0%)</li>
<li>Not Supported: 34532/70481 (49.0%)</li><li>Warnings: 0/70481 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 8.6, Gfx9</summary><ul>

<li>Passed: 33668/66320 (50.8%)</li>
<li>Failed: 0/66320 (0.0%)</li>
<li>Not Supported: 32652/66320 (49.2%)</li><li>Warnings: 0/66320 (0.0%)</li>
</ul></details>


</ul></details>

**Review by nhaehnle — APPROVED** [review]
Thanks!

**Pull request merged**